File: app/src/main/java/com/games/memorygame/network/MemoryGameService.java
package com.themobilecompany.memorygame.network;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;


public interface MemoryGameService {

    @GET("/services/rest/?method=flickr.photos.search&page=1")
    Observable<FlickrResponse> getPhotos(@Query("per_page") int perPage,
                                         @Query("text") String theme);

}

File: app/src/main/java/com/games/memorygame/network/MemoryGameService.kt
package com.games.memorygame.network

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query


interface MemoryGameService {

    @GET("/services/rest/?method=flickr.photos.search&page=1")
    fun getPhotos(@Query("per_page") perPage: Int,
                  @Query("text") theme: String): Observable<FlickrResponse>

}
